package.json

```diff
@@ -34,7 +34,7 @@
     "start-prod": "yarn build-prod && yarn serve-static",
     "start-examples": "ws -d examples/ -s index.html -p 4242",
     "start-docs": "docsify serve ./docs-user",
-    "test": "cross-env NODE_ENV=test jest",
+    "test": "cross-env NODE_ENV=test NODE_OPTIONS=--experimental-worker jest",
```

Comment:
This env variable isn't needed in node v11. v11 isn't a LTS version but maybe we can still upgrade to it because we're not running a server with it?